openverse_catalog/dags/data_refresh/refresh_popularity_metrics_task_factory.py

```diff
@@ -29,20 +30,28 @@ def create_refresh_popularity_metrics_task_group(media_type: str):
     media_type:  the type of record to refresh
     """
     with TaskGroup(group_id=GROUP_ID) as refresh_all_popularity_data:
-        # Update the popularity metrics table, adding any new popularity metrics
-        # and updating the configured percentile.
-        update_metrics = operators.update_media_popularity_metrics(
-            DB_CONN_ID,
-            media_type=media_type,
+        metrics_doc = (
+            "Updates the popularity metrics table, adding any new "
+            "popularity metrics and updating the configured percentile."
+        )
+        update_metrics = PythonOperator(
+            task_id=UPDATE_MEDIA_POPULARITY_METRICS_TASK_ID,
+            python_callable=sql.update_media_popularity_metrics,
+            op_args=[POSTGRES_CONN_ID, media_type],
+            doc=metrics_doc,
         )
```

Comment:
Would it be OK to define the doc in-line? Or is this separated out into a variable to give it more space before having to start a new line? 😅 
```suggestion
        update_metrics = PythonOperator(
            task_id=UPDATE_MEDIA_POPULARITY_METRICS_TASK_ID,
            python_callable=sql.update_media_popularity_metrics,
            op_args=[POSTGRES_CONN_ID, media_type],
            doc=(
                "Updates the popularity metrics table, adding any new "
                "popularity metrics and updating the configured percentile."
            )
        )
```

Reply:
Yeah no reason, I liked the formatting a little better with it separated out but could go either way. Updated